### DISCUSSION TITLE
Linux mobile support

### DISCUSSION THREAD

**Discussion by fairking** [wailsapp/wails#2366] [Q&A] [answered]
Hi everyone.

I didn't find any answers on the internet, but, before I can dive in, I just would like to ask if anyone tried to create and run the Wails based app on [Ubuntu Touch](https://ubuntu-touch.io/) on mobile devices? Or any other Linux mobile distros.

Particularly I am interested in touch support. I guess I am going to use [Quasar](https://github.com/sgosiaco/wails-template-quasar-ts) as a front end, as it is very mobile friendly.

I am also interested in hardware support, not sure how it is going to work with Camera, NFC, File storage.

Please let me know if anyone had a chance to create such apps. Is it hard? Are there any caveats?

**Comment by leaanthony** [comment, marked as answer]
You should check out https://gitlab.com/dedSyn4ps3/wails-pinephone?s=09

**Reply by fairking** [reply]
Wow, thanks. That would help me a lot 👍
